utils/_context/_scenarios/__init__.py

```diff
@@ -363,6 +363,17 @@ def all_endtoend_scenarios(test_object):
         scenario_groups=[ScenarioGroup.APPSEC],
     )
 
+    iast_deduplication = EndToEndScenario(
+        "IAST_DEDUPLICATION",
+        weblog_env={
+            "DD_IAST_ENABLED": "true",
+            "DD_IAST_DEDUPLICATION_ENABLED": "true",
+            "DD_IAST_REQUEST_SAMPLING": "100",
+        },
+        doc="Source code vulnerability standalone mode (APM opt out)",
```

Comment:
`appsec_enabled` is defaulted to `true`. So if i understand correctly your doc, you should add `appsec_enabled=False`

Reply:
I made a copy paste mistake😅 , I've updated the doc